ozaria/site/components/interactive/PageInteractive/draggableOrdering/index.vue

```diff
@@ -0,0 +1,127 @@
+<script>
+  import BaseInteractiveTitle from '../BaseInteractiveTitle'
+  import OrderingSlots from './OrderingSlots'
+
+  export default {
+    components: {
+      'base-interactive-title': BaseInteractiveTitle,
+      'ordering-slots': OrderingSlots
+    },
+
+    props: {
+      interactive: {
+        type: Object,
+        required: true
+      },
+
+      introLevelId: {
+        type: String,
+        required: true
+      },
+
+      interactiveSession: {
+        type: Object,
+        default: undefined
+      },
+
+      courseInstanceId: {
+        type: String,
+        default: undefined
+      }
+    },
+
+    data () {
+      return {
+        draggableGroup: Math.random().toString(),
+
+        promptSlots: [
+          { id: 1, text: 'one option' },
+          { id: 2, text: 'two option' },
+          { id: 3, text: 'three option' },
+          { id: 4, text: 'four option' }
+        ],
+
+        answerSlots: []
+      }
+    }
+  }
+</script>
+
+<template>
+  <div class="insert-code-container">
+    <base-interactive-title
+      :interactive="interactive"
+    />
+
+    <div class="prompt-row">
+      <ordering-slots
+        v-model="promptSlots"
+
+        :draggable-group="draggableGroup"
+
+        class="ordering-column"
+      />
+
+      <ordering-slots
+        v-model="answerSlots"
+        :num-slots="4"
```

Comment:
Looks like ordering slot doesnt use value if there is num-slots, do we need to send answerSlots in this case?

Reply:
Great call out. Simplified the interface.